FRENCH/src/ch05-03-method-syntax.md

```diff
@@ -2,7 +2,7 @@
 ## Method Syntax
 -->
 
-## La syntaxe de méthode
+## La syntaxe des méthodes
```

Comment:
What is wrong with `la syntaxe de méthode`?
I find this term better than `la syntaxe de la méthode` or `la syntaxe des méthodes` which sound like this is a syntax linked it one or several specific methods, whereas `la syntaxe de méthode` makes it clear that it simply refers a specific syntax.

Reply:
I've found it much more natural this way (and sorry, but I'm still).

Don't you agree that we talk about a syntax that is applied on all methods ?

*(and at first i've changed to `la syntaxe de la méthode` before choose `la syntaxe des méthodes`)*

Reply:
Let's go with `la syntaxe des méthodes`.

Reply:
Awesome ! thank you, joel ! 👍